content/swift/concepts/sets/sets.md

```diff
@@ -83,6 +83,6 @@ print("There are \(band.count) players in the band.")
 | `.removeAll()`           | Remove every single value from a set                                                               |
 | `.contains()`            | Check whether an item exists within the set                                                        |
 | `.intersection()`        | Populates a new set of elements with the overlapping elements of two sets                          |
-| `.union()`               | Populates a new set by taking all the values from two sets and combining them                      |
+| `.union()`               | Returns a new set by taking all the values from two sets and combining them                        |
```

Comment:
Let's restore this file since [this PR](https://github.com/Codecademy/docs/pull/1118/files) is going to remove this entire section